security/mk-cacerts.sh

```diff
@@ -85,18 +85,16 @@ IMPORTED=('null')
 for FILE in certs/*.crt; do
     SUBJECT=$(openssl x509 -subject -noout -in "$FILE")
```

Comment:
i did a test
seems two crts got the same subject
-  certs/cert123.crt
-  certs/cert29.crt
subject=C = ES, CN = Autoridad de Certificacion Firmaprofesional CIF A62634068

Reply:
Yes correct, that is a valid duplicate, the Mozilla cert query returns that same Certificate twice